include/mlboard_logger.hpp

```diff
@@ -0,0 +1,200 @@
+/**
+ * @file mlboard_logger.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of the mlbaord logger callback function.
+ */
+#ifndef ENSMALLEN_CALLBACKS_MLBOARD_LOGGER_HPP
+#define ENSMALLEN_CALLBACKS_MLBOARD_LOGGER_HPP
+
+#include <ensmallen.hpp>
+#include <mlboard/core.hpp>
+#include <mlboard/mlboard.hpp>
+
+namespace ens {
+
+/**
+ * Mlboard Logger function, based on the EndEpoch callback function.
+ */
+class MlboardLogger
+{
+ public:
+  /**
+   *
+   * @param output Filewriter object to log the metrics.
```

Comment:
Hey, Can you add description to this function or remove the top line? Also do the same for other funtion.